docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,442 @@
 
+    Current node cluster state (term, version) pair is (4, 12)
 
-.. WARNING::
+    Do you want to proceed?
 
-  Execution of this command can lead to arbitrary data loss. Only run this tool
-  if you understand and accept the possible consequences and have exhausted all
-  other possibilities for recovery of your cluster.
+    Confirm [y/N] n
 
-The sequence of operations for using this tool are as follows:
+* Run ``crate-node unsafe-bootstrap`` on the second remaining node
+  and answer ``n`` at the confirmation step.
 
-* Make sure you have really lost access to every one of the master-eligible nodes
-  in the cluster, and they cannot be repaired or recovered by moving their data
-  paths to healthy hardware.
+.. code-block:: console
 
-* Start a new cluster and verify that it is healthy. This cluster may comprise
-  one or more brand-new master-eligible nodes, or may be an unsafely-bootstrapped
-  cluster formed as described above.
+    sh$ node_2$ ./bin/crate-node unsafe-bootstrap
 
-* Stop all remaining data nodes.
+    Current node cluster state (term, version) pair is (5, 3)
 
-* On each data node, run the ``crate-node detach-cluster`` tool as shown
-  below. Verify that the tool reported Node was successfully detached from
-  the cluster.
+    Do you want to proceed?
 
-* If necessary, configure each data node to discover the new cluster.
+    Confirm [y/N] n
 
-* Start each data node and verify that it has joined the new cluster.
+Since the second node has a greater term it has a fresher cluster state, so it
+is better to unsafely bootstrap the cluster using the second node.
 
-* Wait for all recoveries to have completed, and investigate the data in the
-  cluster to discover if any was lost during this process.
+* Run ``crate-node unsafe-bootstrap`` on the second remaining node and answer
+  ``y`` at the confirmation step.
 
-The message Node was successfully detached from the cluster does not mean that
-there has been no data loss, it just means that tool was able to complete its job.
+.. code-block:: console
 
-Parameters
-----------
+    sh$ node_2$ ./bin/crate-node unsafe-bootstrap
 
-`repurpose`
-  Delete excess data when a node’s roles are changed.
+    Current node cluster state (term, version) pair is (5, 3)
 
-`unsafe-bootstrap`
-  Specifies to unsafely bootstrap this node as a new one-node cluster.
+    Do you want to proceed?
 
-`detach-cluster`
-  Specifies to unsafely detach this node from its cluster so it can join a
-  different cluster.
+    Confirm [y/N] y
+    Master node was successfully bootstrapped.
 
-`--ordinal <Integer>``
-  If there is more than one node sharing a data path then this specifies which
-  node to target. Defaults to 0, meaning to use the first node in the data path.
+Before running the unsafe-bootstrap tool
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-`-C <KeyValuePair>`
-  Configures a setting.
+* Make entirely sure you have lost access to at least half of the
```

Comment:
can you reformat this list so that you're not starting the section with a list

also the list items could be shortened a little/rephrased

same with the other big cautionary list